charts/cluster-autoscaler/values.yaml

```diff
@@ -200,6 +200,9 @@ extraEnvSecrets: {}
 # envFromConfigMap -- ConfigMap name to use as envFrom.
 envFromConfigMap: ""
 
+# envFromConfigMapOptional -- Whether or not the ConfigMap in envFrom is optional
+envFromConfigMapOptional: true
```

Comment:
Is there a good reason to change the default behaviour here? 

Unless I'm mistaken the current behaviour prior to this is for the envFromConfigMap to be required if referenced.

Reply:
What I've found handy about using the "optional" property is that it allows you to use the same manifest for different environments where the ConfigMap only comes into play if it's applied to the cluster. For example, where one environment may have a proxy, which requires a ConfigMap with the proxy env variables, but another environment may not have a proxy so the ConfigMap isn't needed or applied.

Reply:
Thanks for the extra detail, I'm not against adding the option, my question was more around the defaulting of the new option to true, which changes the existing behaviour. I'd rather preserve the current behaviour but allow those who want to change it to override it when adding new functionality.